AUTHORS

```diff
@@ -0,0 +1,16 @@
+The authors, designers, and main contributors to lean-smt are listed below.
+lean-smt's copyright is held by these individuals and the affiliated
+institutions at the time of their contributions (note that some authors have
+had more than one affiliated institution). See the file LICENSE for details on
+the copyright and licensing of lean-smt.
+
+The developers and authors of lean-smt are:
+
+Current:
+  Haniel Barbosa, Universidade Federal de Minas Gerais
+  Tomaz Gomes Mascarenhas, Universidade Federal de Minas Gerais
```

Comment:
@tomaz1502 would you prefer to mention your full name (as shown above) or just your first and last name?